Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'fast_gettext', '~> 0.8'
 gem 'gettext_i18n_rails', '~> 1.0'
 gem 'i18n', '~> 0.6.4'
 gem 'turbolinks', '~> 2.5'
+gem 'logging', '>= 1.8.0', '< 2.0.0'
```

Comment:
Is there anything in 2.0.0 that's a problem?  A brief scan of the changelog looks okay.  I ask as it's out, so I don't want to start on the back foot, and we're already using it in our bundles due to other gems using `~> 2.0`.